flaps/flaps.go

```diff
@@ -114,6 +114,17 @@ func (f *Client) Start(ctx context.Context, machineID string) (*api.MachineStart
 	return out, nil
 }
 
+func (f *Client) TryWait(ctx context.Context, machine *api.Machine, state string, tries int) (err error) {
```

Comment:
Were you experiencing a specific issue that resulted in this change?

Reply:
Yes, in one of the tests, machine took more than `30s` to come up.

Reply:
`run.go` anticipates it, too: https://github.com/superfly/flyctl/blob/7fe7fd5b4d0f21e18e950a344675b32b3978b447/internal/command/machine/run.go#L298 (may be `update` should use this, instead).

Reply:
Done